### PR TITLE
Fixed broken cleanup phase

### PR DESCRIPTION
**Pull request by DunderRoffe**

### CODE REVIEW

File: common/softwarecontainer-common.cpp [hunk 1]
```diff
@@ -157,10 +157,6 @@ SignalConnectionsHandler::~SignalConnectionsHandler()
     }
 }
 
-bool operator&& (ReturnCode lhs, ReturnCode rhs) {
-    return lhs == rhs;
-}
```
Comment (tobsan) [inline common/softwarecontainer-common.cpp:162]: Why is this being removed?

File: common/softwarecontainer-common.h [hunk 2]
```diff
@@ -61,8 +61,6 @@ enum class ReturnCode
     SUCCESS
 };
 
-bool operator&& (ReturnCode lhs, ReturnCode rhs);
```
Comment (tobsan) [inline common/softwarecontainer-common.h:64]: And why is this being removed?

**Review by tobsan — COMMENTED** [review]
Looks good, just wondering about the ReturnCode stuff that is being removed.

File: common/softwarecontainer-common.cpp (hunk 1 above)
Comment (DunderRoffe) [reply common/softwarecontainer-common.cpp:162]: This operator makes no sense.

``` cpp
ReturnCode::FAILURE && ReturnCode::FAILURE
```

does **not** equal

``` cpp
ReturnCode::FAILURE == ReturnCode::FAILURE
```

Use isError() or isSuccess() instead

File: common/softwarecontainer-common.h (hunk 2 above)
Comment (DunderRoffe) [reply common/softwarecontainer-common.h:64]: See above

File: common/softwarecontainer-common.cpp (hunk 1 above)
Comment (tobsan) [reply common/softwarecontainer-common.cpp:162]: OK!

**Pull request merged**